include/quda_define.h.in

```diff
@@ -251,36 +265,51 @@
 #endif
 
 /**
- * @def QUDA_ORDER_FP
- * @brief This macro sets the data ordering for Wilson, gauge
- * (recon-8/9) and clover fixed-point fields
+ * @def QUDA_ORDER_DOUBLE
+ * @brief This macro sets the data ordering for double precision fields
+ */
+#define QUDA_ORDER_DOUBLE @QUDA_ORDER_DOUBLE@
+
+#ifdef __cplusplus
```

Comment:
Similar comment as the i2f carveout, applies below as well...

Reply:
d247829a5cae3d13d5fdc747fc46d77790520673